docs/docs/migrate-from-v1/README.md

```diff
@@ -0,0 +1,263 @@
+# Migrate from Pipedream v1
```

Comment:
I'd recommend using the language from Tod's Slack announcement here, and then add links to the relevant pages like Python, Go, etc. from there:

We have re-imagined the UX from the ground up, made the product much easier to use and have improved performance.  In addition, we are introducing powerful new features including:

- Improved builder to test and iterate without impacting live workflows
- Multi-language support including Node, Python, Golang & bash
- Granular testing including the ability to test individual steps and more
- Multiple triggers are now supported per workflow
- Improved forms for easier configuration and streamlined building